src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -436,7 +436,7 @@ void shouldReturnListOfAllWalletsByName() throws Exception {
         List<WalletDTO> listOfWalletsDTO = createListOfWalletsDTO();
         List<WalletDTO> foundedWalletsDTO = List.of(new WalletDTO(ID_OF_WALLET_2, NAME_OF_WALLET_2, CREATION_DATE_OF_WALLET_2));
         given(walletService.getAll()).willReturn(listOfWalletsDTO);
-        given(walletService.findByName(walletNameSearched)).willReturn(foundedWalletsDTO);
+        given(walletService.findAllByName(walletNameSearched)).willReturn(foundedWalletsDTO);
 
         // then
         mockMvc.perform(get("/api/wallet/find/{name}", walletNameSearched)
```

Comment:
API paths should not contains verbs,  `/api/wallet/{name}`  I think would be better.

Reply:
you can change for example: /api/wallet/list/{name}

Reply:
> you can change for example: /api/wallet/list/{name}

```markdown
2.8 Filter Operations

While building RESTful APIs, we will come across multiple requirements where we need collection of resources with

    Sorting
    Filtering our certain results from the collection.
    Limited results based on certain conditions 

Do not create new API for these requirements.Use query parameters to achieve all these results.

http://www.javadevjournal.com/products?catalog=Online
http://www.javadevjournal.com/products?region=USA&brand=dummy&sort=new

```

**source**
https://www.javadevjournal.com/spring/rest-resource-naming/